camel/src/main/java/com/github/theprez/manzan/routes/event/WatchMsgEvent.java

```diff
@@ -15,29 +18,31 @@ public class WatchMsgEvent extends ManzanRoute {
     private final String m_schema;
     private final String m_sessionId;
     private final ManzanMessageFormatter m_formatter;
+    private final String m_socketIp = "0.0.0.0";
+    private final String m_socketPort = "8080";
```

Comment:
Will we need to make this configurable? If yes, can look into doing this in a separate PR.